src/jdk.jlink/share/classes/jdk/tools/jlink/resources/jlink.properties

```diff
@@ -157,3 +160,6 @@ warn.provider.notfound=No provider found for service specified to --suggest-prov
 no.suggested.providers=--bind-services option is specified. No additional providers suggested.
 suggested.providers.header=Suggested providers
 providers.header=Providers
+
+runtime.link.info=Linking based on the current run-time image.
+runtime.link.jprt.path.extra=(run-time image)
```

Comment:
Missing newline at last line.

Reply:
... still missing...

Reply:
Actually, this is a bit strange. I thought jcheck would look for missing newline at EOF, and that properties files were included in the check nowadays. I'll need to check this out.

Reply:
I did some testing and it turns out that this is indeed not checked. I believe this is a miss in the Skara reimplementation of jcheck. I've opened https://bugs.openjdk.org/browse/SKARA-2265 to track this.

Nevertheless, it would be good if you could fix this. :)

Reply:
Sure.

Reply:
Should be fixed now.